lib/src/meta.dart

```diff
@@ -324,14 +324,9 @@ typedef FlattenEachCallback = dynamic Function(
 /// ```
 void flattenEach(GeoJSONObject geoJSON, FlattenEachCallback callback) {
   try {
-    geomEach(geoJSON, (currentGeomObject, featureIndex, featureProperties,
-        featureBBox, featureId) {
-      if (currentGeomObject is GeometryCollection) {
```

Comment:
I removed this if clause, because it's already handled by that geomEach now takes a GeometryType?, which cannot be a GeometryCollection.

CC @baparham